### ISSUE TITLE
Documentation overhaul

### ISSUE THREAD

**Issue by scriptingosx** [Installomator/Installomator#790]
We have made some progress in improving the documentation, especially @Theile 's series on how to build labels. However, there is much work that needs to be done.

This issue will list all the pieces, we can create sub-issues or PRs for the details:

- [ ] Clean up main ReadMe and distribute most of the information into individual wiki pages
- [ ] Variables (list and describe all variables)
- [ ] How to set variables 
    - [ ] from the command line and in scripts
    - [ ] in Jamf
- [ ] Setup Instructions with management systems and sample scripts (where necessary
    - [ ] Jamf (update and transfer @mischavdbent 's [post](https://scriptingosx.com/2020/06/using-installomator-with-jamf-pro/))
    - [ ] Use SwiftDialog for progress with Jamf Pro Self service actions
    - [ ] Mosyle 
    - [ ] Intune
    - [ ] WorkspaceOne
    - [ ] Addigy
    - [ ] ?
- [ ] FAQ
    - [ ] move existing FAQ
    - [ ] explain issues with new versions
    - [ ] new versions and Jamf Patch management
    - [ ] how to add variables in Jamf Pro policy

**Comment by acodega** [comment]
The "How Installomator works" wiki page is blank, it's a place holder right now.

**Comment by Theile** [comment]
I would add, that maybe differentiate documentation between what is documented in the script and what is in the wiki, so it should not be double, but support each other.

Maybe Adam has some great feelings for what are the most asked questions, but to me it loks like people have a hard time of figuring out where to start with this, what the great thing is, and what Installomator is not.

We should also link to the MacSysAdmin video this year, as well as the JNUC one, when that is publicly available.

**Comment by Theile** [comment]
I think it could be time to consider to move the `# NOTE: How labels work`-section to wiki, so only the variables are in the script.

**Comment by scriptingosx** [comment]
I agree that everything that is in the script should be in the wiki. But I don't want to remove it from the script. For some people, finding the script (possibly an older version) in their Jamf Pro (or other management system) might be their first and possibly only encounter with Installomator
